rudder-web/src/main/resources/rudder-vhost-ssl.conf

```diff
@@ -46,5 +46,12 @@
 
   # MSIE 7 and newer should be able to use keepalive
   BrowserMatch "MSIE [17-9]" ssl-unclean-shutdown
+  
+  #set "secure" and "HttpOnly" for all cookies without these pragma
+  Header edit Set-Cookie "(?i)^((?:(?!;\s?HttpOnly).)+)$" "$1; HttpOnly"
+  Header edit Set-Cookie "(?i)^((?:(?!;\s?secure).)+)$" "$1; secure"
```

Comment:
Not sure if mod_header is enabled by default nor what should be done to ensure that.